lib/config.js

```diff
@@ -70,7 +70,11 @@ exports.load = function(fileName, currentEnv) {
   } else if(config['default']) {
     out.setCurrent(config['default']);
   } else if(config.env) {
-    out.setCurrent(config.env);
+    if (config.env.ENV) {
```

Comment:
Good question, the config are actually a mix of everything. This is due to the nature of throughpassing to the underlying services. So as multipleStatements comes from `node-mysql` and note `db-migrate`. I prefer camelCase in code, but configurations are mostly a different thing where I prefer snake case. In this case I think it wouldn't be that wrong to use snake case, as from the following points.

- This tool is mainly used by developers and operations executives with the knownledge to not beeing confused by this.
- Most of the used drivers use camel case for their config.